script/upgrades/dependencies.json

```diff
@@ -1,5 +1,9 @@
 {
-  "42220": {},
+  "42220": {
+    "BridgedUSDC": "0xEB466342C4d449BC9f53A865D5Cb90586f405215",
```

Comment:
sucks that this contract is not verified

Reply:
👍 agreed.